code-examples/quarkus-service-registry-quickstart/consumer/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[Question]
What is the benefit of having separate consumer and producer if we run them separately. 
From configuration point of view we can then have single application.properties etc.

That will also simplify our SBO integration in later phase.
No strong opinion on this but considering couple factors:

- Maintenance of the example/quarkus version updates will be easier 
- Testing and automation
- Less code to maintain and chance to diverge.
- More aligned with Kafka example as this is single module

Drawbacks:

- Obviously not canonically correct (most the apps doesn't produce and consume at the same time)
- Less of quickstart for users (producting/consuming flags will be mixed together)

Reply:
I think the idea behind this is exactly what you mention as a drawback, to give users the simplest but realistic quickstart. As you said, in a real environment the producer and the consumer will be different applications. I don't have a strong opinion here either. @bibryam thoughts?

Reply:
Agree. Worth to mention:
This drawback can be addressed by comments in application properties. 
Having all config visible in a single property file is actually. 
There is also the second argument which is eventual confusion - we have 2 apps but they run as a single process that might also make the openshift deployment (SBO) guide a little bit confusing. 

Using single module will cut code by half (considering that there is no real owner of those examples after we deliver them that is good thing to have). I will also allow us to build up continuous tutorial - like Kafka guide that we have now can be adjusted to service registry scenario and that will require making changes in Kafka guide itself.

Reply:
I understand your points and I think they make sense, I will leave the decision to Bilgin. @bibryam What do you prefer?